Source/scrollrt.cpp

```diff
@@ -698,7 +698,7 @@ void __fastcall scrollrt_draw_lower(int x, int y, int sx, int sy, int a5, int so
 // 69CF94: using guessed type int cel_transparency_active;
 // 69CF98: using guessed type int level_piece_id;
 
-void __fastcall scrollrt_draw_clipped_dungeon(char *a1, int sx, int sy, int a4, int a5, int a6)
+void __fastcall scrollrt_draw_clipped_dungeon(char *pBuff, int x, int y, int sx, int sy, int me_flag)

```

Comment:
@galaxyhaxz mentioned that it should be:
```cpp
void __fastcall scrollrt_draw_clipped_dungeon(char *pBuff, int sx, int sy, int dx, int dy, int me_flag)
```
But since a2 and a3 are being used to access dDead/dFlags/dObject I don't see how this makes sense.

I settled on x/y instead of dx/dy to keep it consistent with what is already there for all other functions, but I could be convinced to change it threw out the file.

Reply:
The assert string from debug makes it pretty clear `"(DWORD)sx < MAXDUNX"`, the confusion was that a2/a3 refers to dungeon tiles and a4/a5 refers to screen pixels. I thought it would be dungeonX/Y screenX/Y but they seem to mean startX/Y destinationX/Y.
This is now corrected.